src/pages/index.md

```diff
@@ -29,6 +25,27 @@ GraphQL has several advantages over REST and other APIs:
 -  **Backwards Compatibility** - Preserve existing APIs while gradually adopting new ones
 -  **Extensibility** - Customize and extend all of your APIs directly in the gateway without making changes to the API source
 
-## Example use case
+### Getting Started
+
+[API Mesh](gateway/overview.md)
+
+Get started with the API Mesh Overview.
+
+<DiscoverBlock slots="heading, link, text"/>
+
+### Guides
+
```

Comment:
Add a line saying something like 

Review the following guides to learn more about the API Mesh.

and adjust the DiscoverBlock as necessary